test011.c: 函数 `test000' :
test011.c:10: エラー: register 宣言されている `a' のアドレスが参照されています.
test011.c:9: エラー: `a' はここで register 宣言されています.
test011.c: 函数 `test001' :
test011.c:19: エラー: 単項の `&' が register 宣言されている `s' に指定されています.
test011.c:18: エラー: `s' はここで register 宣言されています.
test011.c: 函数 `test002' :
test011.c:28: エラー: 単項の `&' が register 宣言されている `u' に指定されています.
test011.c:27: エラー: `u' はここで register 宣言されています.
